resources/event/event.controller.js

```diff
@@ -97,17 +97,19 @@ const findByDate = async (startDate, endDate, userCity, userCountry, eventType)
 
 const createEvent = async (req, res) => {
   const { file } = req;
+
   if (file) {
     try {
-      const { url } = await imageUploader(file);
-      req.body.imageUrl = url;
+      const response = await imageUploader(file);
+      req.body.imageUrl = response.url;
     } catch (e) {
-      res.status(400).json({ message: 'Could not upload image' });
+      res.status(400).json({ message: 'Could not create event.' });
     }
-  } else req.body.url = "";
+  } else req.body.imageUrl = "";
```

Comment:
Since the above, if has curly brace could you complete the else also to be consistent